src/widget/wtrackmenu.cpp

```diff
@@ -471,14 +471,11 @@ void WTrackMenu::updateMenus() {
 
         // We load a single track to get the necessary context for the cover (we use
         // last to be consistent with selectionChanged above).
-        TrackPointer last = trackPointers.last();
-        CoverInfo info;
-        info.source = last->getCoverInfo().source;
-        info.type = last->getCoverInfo().type;
-        info.hash = last->getCoverHash();
-        info.trackLocation = last->getCoverInfoWithLocation().trackLocation;
-        info.coverLocation = last->getCoverInfoWithLocation().coverLocation;
-        m_pCoverMenu->setCoverArt(info);
+        TrackPointer pTrack = trackPointers.last();
+
+        m_pCoverMenu->setCoverArt(
+                pTrack->getCoverInfoWithLocation());
+        m_pMetadataMenu->addMenu(m_pCoverMenu);
```

Comment:
Is this last line an accident? This wasn't in the previous code.

Reply:
Oh, good catch. Maybe I forgot to remove this line after copying some code while resolving merge conflicts.

Reply:
I have removed the wrong line from the original commit and force pushed the branch.